src/utils/methods.js

```diff
@@ -0,0 +1,21 @@
+export function validateEmail(value) {
+  if (!value) {
+    return 'Email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(value)) {
+    return 'Email is not valid';
+  }
+}
```

Comment:
The function `validateEmail` does not return anything if the email is valid. Consider returning a success message or `null` to indicate that the email is valid.